src/App.js

```diff
@@ -42,6 +42,11 @@ export default function App() {
   const [allTransactions, setAllTransactions] = useState([]);
   const [lastUpdate, setLastUpdate] = useState('');
   const [prependFilenameWithAcctNames, setPrependFilenameWithAcctNames] = useState(true);
+  const [divisorPower, setDivisorPower] = useState(9); // ONEDAY: read from localStorage
+  const divisorPowerOptions = [[0, 9].map((x) => ({ value: x, label: x ? `1e${x}` : 1 }))].flat();
+  const [decimalPlaces, setDecimalPlaces] = useState(6); // ONEDAY: read from localStorage
```

Comment:
For the future you can find online a few useLocalStorage hook that can help make the job easy

Reply:
Thanks! I used https://usehooks.com/useLocalStorage/